### DISCUSSION TITLE
Question: Mac Catalyst

### DISCUSSION THREAD

**Discussion by rehatkathuria** [pointfreeco/swift-parsing#308] [General]
Any plans to support Mac Catalyst as a platform?

**Comment by mbrandonw** [comment]
Hi @rehatkathuria, I think the library already supports Mac Catalyst just fine. What problems are you having?

**Comment by rehatkathuria** [comment]
I'm seeing a module not found error. Could it be related to `.macCatalyst` not being defined as a platform in `Package.swift`?

**Comment by mbrandonw** [comment]
I just created a fresh Mac Catalyst project and was able to add parsing just fine. Can you share a small sample project that demonstrates the problem?

**Comment by mbrandonw** [comment]
I'm going to transfer this to a discussion since it is not clear that it's an issue with the library. If you can produce a sample project we will look into it.
